lib/plugins/filter/after_post_render/external_link.js

```diff
@@ -11,6 +11,7 @@ function externalLinkFilter(data) {
 
   if (typeof EXTERNAL_LINK_POST_CONFIG === 'undefined') {
     if (typeof config.external_link === 'undefined' || typeof config.external_link === 'object'
+      // Deprecated: config.external_link boolean option will be removed in future
```

Comment:
In fact we could remove the boolean value support in Hexo 5.0.0, as it was introduced in Hexo 4.0.0.